test/components/views/rooms/SendMessageComposer-test.tsx

```diff
@@ -136,13 +147,14 @@ describe('<SendMessageComposer/>', () => {
     });
 
     describe("functions correctly mounted", () => {
-        const mockClient = MatrixClientPeg.matrixClient = createTestClient();
-        const mockRoom = mkStubRoom('myfakeroom') as any;
+        const mockClient = createTestClient();
+        jest.spyOn(MatrixClientPeg, 'get').mockReturnValue(mockClient);
+        const mockRoom = mkStubRoom('myfakeroom', 'myfakeroom', mockClient) as any;
         const mockEvent = mkEvent({
             type: "m.room.message",
             room: 'myfakeroom',
             user: 'myfakeuser',
-            content: "Replying to this",
+            content: { "m.text": "Replying to this" },
```

Comment:
`m.room.message` has a somewhat strict schema at present:
```suggestion
            content: { "msgtype": "m.text", "body": "Replying to this" },
```